inc/fields.class.php

```diff
@@ -329,4 +328,39 @@ public static function updateVisibility($currentValues) {
 
       return $questionToShow;
    }
+
+   /**
```

Comment:
Documentation was copied from `fieldTypeExists()` and is not suitable for this method.